specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json

```diff
@@ -0,0 +1,946 @@
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/SearchResponse"
+            }
+          },
+          "default": {
+            "description": "An error has occurred. Check the response type and/or status code for more details.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SearchResponse": {
+      "description": "Defines the top-level object that the response includes when the request succeeds.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Response"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "queryContext": {
+          "description": "An object that contains the query string that Bing used for the request. This object contains the query string as entered by the user. It may also contain an altered query string that Bing used for the query if the query string contained a spelling mistake.",
+          "$ref": "#/definitions/QueryContext",
+          "readOnly": true
+        },
+        "places": {
+          "description": "A list of local entities such as restaurants or hotels that are relevant to the query.",
+          "$ref": "#/definitions/Places",
+          "readOnly": true
+        },
+        "lottery": {
+          "$ref": "#/definitions/SearchResultsAnswer",
+          "readOnly": true
+        },
+        "searchResultsConfidenceScore": {
+          "readOnly": true,
+          "type": "number",
+          "format": "float"
+        }
+      }
+    },
+    "Response": {
+      "description": "Defines a response. All schemas that return at the root of the response must inherit from this object.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Identifiable"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "readLink": {
+          "description": "The URL that returns this resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "webSearchUrl": {
+          "description": "The URL to Bing's search result for this item.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "potentialAction": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Action"
+          }
+        },
+        "immediateAction": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Action"
+          }
+        },
+        "preferredClickthroughUrl": {
+          "readOnly": true,
+          "type": "string"
+        },
+        "adaptiveCard": {
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "QueryContext": {
+      "description": "Defines the query context that Bing used for the request.",
+      "discriminator": "_type",
```

Comment:
Could we rename the discriminator? Usually, names should follow `camelCase` style.

Reply:
QueryContext is an inherited schema that is used all over the place. I cannot change it.